frontend/www/js/omegaup/ui.ts

```diff
@@ -271,3 +271,26 @@ export function rankingUsername(
     username = formatString(T.virtualSuffix, { username: username });
   return username;
 }
+
+export function updateTheme(): boolean {
```

Comment:
```suggestion
export function updateTheme(): void {
```

`updateTheme` es una acción, entonces debería regresar verdadero / falso si la acción se ejecutó correctamente. aquí el valor de retorno no indica eso (y en los lugares donde se llama no se usa el valor de retorno), entonces es mejor no regresar nada.

Reply:
Listo